src/Adapter/AbstractObjectModelRepository.php

```diff
@@ -95,6 +116,43 @@ protected function getObjectModel(int $id, string $objectModelClass, string $exc
         return $objectModel;
     }
 
+    /**
+     * @param int $id
+     * @param string $objectModelClass
```

Comment:
```suggestion
     * @param class-name<ObjectModel> $objectModelClass
```

see https://phpstan.org/writing-php-code/phpdoc-types#class-string

Reply:
The parameter should probably be renamed to `$objectModelClassName`

Reply:
I didn't know this one, nice!

Reply:
However I have two warnings in my IDE, it doesn't seem to bother CS-fixer nor PHPStan though

![Capture d’écran 2021-08-26 à 13 55 55](https://user-images.githubusercontent.com/13801017/130958585-8bce6f08-7ae9-4896-90ca-93c78c4be542.png)

![Capture d’écran 2021-08-26 à 13 56 51](https://user-images.githubusercontent.com/13801017/130958651-cb3fb46f-268f-4c85-be56-53b23f020e0a.png)

Reply:
Actually, PHPStan does not appreciate it.. that's weird Maybe we don't have a recent enough version?

![Capture d’écran 2021-08-26 à 13 58 06](https://user-images.githubusercontent.com/13801017/130958854-d59d0461-b85f-47d5-be43-ca5c6a419b82.png)

Reply:
It should be `class-string<ObjectModel>` and not `class-name` :thinking: